tests/services/auth.service.test.js

```diff
@@ -106,31 +97,21 @@ describe('Auth Service', () => {
                 email: 'jon.doe@email.com',
                 password: 'password1',
             }
-            const expectedResult = {
+            const stubUser = {
                 id: '1',
                 email: 'jon.doe@email.com',
                 firstName: 'Jon',
                 lastName: 'Doe',
                 password: '123dj4*#&@DJ@941nd',
             }
-            userRepositoryGetUserByEmailStub.returns(expectedResult)
+            userRepositoryGetUserByEmailStub.returns(stubUser)
 
-            const response = authService.verifyPassword(
+            const user = authService.verifyPassword(
                 payload.email,
                 payload.password
             )
 
-            // Question: is there a need to expect steps to hash payload's password and compare against password in db?
-            expect(
-                userRepositoryGetUserByEmailStub.calledOnceWithExactly(
-                    payload.email
-                )
-            ).toBeTruthy()
-            expect(response.id).toBeDefined()
-            expect(response.email).toBeDefined()
-            expect(response.firstName).toBeDefined()
-            expect(response.lastName).toBeDefined()
-            expect(response.password).toBeDefined()
+            expect(user).toEqual(stubUser)
```

Comment:
Usually I have a single assert statement that asserts on the whole object. Otherwise we might have a scenario where we test that an "elephant" has a "trunk" and "4 legs", but forget to assert that it also has a "tail". One day the "tail" might go missing and the tests will still pass.

Better to assert on the whole elephant :-)